qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -209,6 +284,8 @@ def get_qubit_operators(
         if self.solver.supports_aux_operators():
             if isinstance(self.qubit_converter, QubitConverter):
                 aux_ops = self.qubit_converter.convert_match(aux_second_q_operators)
+            elif isinstance(self.qubit_converter, TaperedQubitMapper):
+                aux_ops = self.qubit_converter.map_clifford(aux_second_q_operators)
             else:
                 aux_ops = self.qubit_converter.map(aux_second_q_operators)
```

Comment:
I see this same structure in multiple places in this file. Could you extract this into a private method?